components/team_sidebar/index.ts

```diff
@@ -29,15 +29,19 @@ import {switchTeam, updateTeamsOrderForUser} from 'actions/team_actions.jsx';
 
 import {Preferences} from 'utils/constants';
 
+import {getForceTeamSidebarToBeVisible} from '../../selectors/views/team_sidebar';
```

Comment:
Can you place this by the other `selectors` imports above?

```suggestion
import {getForceTeamSidebarToBeVisible} from 'selectors/views/team_sidebar';
```